IFC4x3/Constants/v/VACUUMDEVICE_1et7nWAsjDZ9dTWZwT6GWo.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="VACUUMDEVICE_1et7nWAsjDZ9dTWZwT6GWo" Name="VACUUMDEVICE" UniqueId="68dc7c60-2b6b-4d8c-99dd-823e9d190832">
-	<Documentation>A mooring devices that uses vacuum suction as the primary method of securing the vessel.</Documentation>
+	<Documentation>Mooring devices that uses vacuum suction as the primary method of securing the vessel.</Documentation>
```

Comment:
'that uses' -> 'that use'. For the sake of consistency. I would prefer singular in these cases since the PDT refers to an occurrence and existing PDTs usually use singular (at least most that I know of).

Reply:
no problem